src/main/resources/edu/cornell/kfs/vnd/document/workflow/CuVendorMaintenanceDocument.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+    <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+        <documentType>
+            <name>PVEN</name>
+            <parent>VEND</parent>
+            <label>Vendor</label>
+            <authorizer>org.kuali.kfs.krad.workflow.authorizer.CfDocumentTypeAuthorizer</authorizer>
+            <helpDefinitionURL>${externalizable.help.url}/default.htm?turl=WordDocuments%2Fvendorpven.htm</helpDefinitionURL>
+            <docSearchHelpURL>${externalizable.help.url}/default.htm?turl=WordDocuments%2Fvendorpven.htm</docSearchHelpURL>
+            <active>true</active>
+            <routingVersion>2</routingVersion>
+            <routePaths>
+                <routePath>
+                    <start name="AdHoc" nextNode="RequiresApproval"/>
+                    <split name="RequiresApproval" nextNode="RequiresVendorTaxIdManager">
+                        <branch name="True">
+                            <role name="Management" nextNode="Initiator"/>
+                            <role name="Initiator" nextNode="JoinRequiresApproval"/>
+                        </branch>
+                        <branch name="False">
+                            <simple name="Do Nothing" nextNode="JoinRequiresApproval"/>
+                        </branch>
+                        <join name="JoinRequiresApproval"/>
+                    </split>
+                    <split name="RequiresVendorTaxIdManager">
+                        <branch name="True">
+                            <role name="Vendor Tax ID Reviewer" nextNode="JoinRequiresVendorTaxIdManager"/>
```

Comment:
Usually the node names don't have spaces. As long as this is not causing any issues, I am fine with it like this.